### ISSUE TITLE
Fix location hierarchy

### ISSUE THREAD

**Issue by caywood** [OpenMobilityData/OpenMobilityData-Public#425]
http://transitfeeds.com/admin/locations/

The "create a location" flow doesn't seem to work as there's no button to create a location.

![screen shot 2019-02-12 at 3 36 03 pm](https://user-images.githubusercontent.com/11506462/52667012-c2102980-2edd-11e9-9c50-9408871baec9.png)

Also the "Save Parent location" button is broken and only lets you parent under a sibling, which is irrecoverable through the Admin UI so don't press it.

![screen shot 2019-02-12 at 3 50 21 pm](https://user-images.githubusercontent.com/11506462/52667087-f683e580-2edd-11e9-9883-666915bb719e.png)

@canales proposes to import geonames (CC-BY licensed) into the database and remove the Locations UI. 

@HendX please weigh in on whether this makes sense or if we're missing something obvious.

**Comment by ethanpooley** [comment]
@caywood Our Google API key, which we switched to during the initial takeover, wasn't working for the geocoding feature that is relied on when creating locations. I've changed that back temporarily to the key that was being used before, but we'll have to sort that out soon.

Regarding the "Save Parent Location" feature, it seems the drop-down is populated with all siblings and all parents/grandparents/etc. up the chain. This facilitates moving a location up or down the hierarchy in steps, as well as laterally by going "up one and over". Clunky to understand, but kind of elegant in its minimal UI needs.

The two problems with it are a lack of explanation and the potential for confusion/disaster at the top level. Since the continents have no parent (they are each a top-level parent themselves), if you move one down you cannot move it back up. Best fixed in this system by adding a "Planet" level. But unnecessary if we're going to move away from this entire approach.
